easybuild/easyconfigs/a/Amber/Amber-16-iomkl-2016.09-GCC-4.9.3-2.25-AmberTools-16-patchlevel-5-14-CUDA.eb

```diff
@@ -0,0 +1,36 @@
+# Automatically converted from Amber-16-iomkl-2016.09-CUDA.eb
+# by /gpfs/sysapps/bham/scripts/easybuild/recursive_build_with_new_toolchain.py
+#
+# Based on https://raw.githubusercontent.com/hpcugent/easybuild-easyconfigs/master/easybuild/easyconfigs/a/Amber/Amber-14-intel-2016a-AmberTools-15-patchlevel-13-13.eb
+#
+name = 'Amber'
+version = '16'
+ambertools_ver = '16'
+# Patch levels from http://ambermd.org/bugfixes16.html and http://ambermd.org/bugfixesat.html
+patchlevels = (5, 14)  # (AmberTools, Amber)
+versionsuffix = '-AmberTools-%s-patchlevel-%s-%s-CUDA' %(ambertools_ver, patchlevels[0], patchlevels[1])
+
+homepage = 'http://ambermd.org/amber.html'
+description = """Amber (originally Assisted Model Building with Energy
+ Refinement) is software for performing molecular dynamics and structure
+ prediction."""
+
+toolchain = {'name': 'iomkl', 'version': '2016.09-GCC-4.9.3-2.25'}
+toolchainopts = {'usempi': True}
+
+sources = [
+    'Amber%(version)s.tar.bz2',
+    'AmberTools%s.tar.bz2' % ambertools_ver,
+]
+
+dependencies = [
+    ('CUDA', '7.5.18', '', ('iccifort', '2016.3.210-GCC-4.9.3-2.25')),
```

Comment:
@edmondac any particular reason for hardcoding a (sub)toolchain here as opposed to using the `dummy` version of CUDA (see existing `Amber-14-intel-2016a-AmberTools-15-patchlevel-13-13.eb`)?